core/typedefs.h

```diff
@@ -67,7 +67,9 @@ static_assert(__cplusplus >= 201703L);
 
 // Should always inline, except in dev builds because it makes debugging harder.
```

Comment:
```suggestion
// Should always inline, except in dev builds because it makes debugging harder,
// or `size_enabled` builds where inlining is actively avoided.
```